compose/svc.Dockerfile

```diff
@@ -36,54 +36,49 @@ RUN luajit ./tests/test.lua
 # nxsearch-svc image
 #
 
-# OpenResty on Debian (11.x -- Bullseye)
```

Comment:
Can you please retain this comment? I keep forgetting what numeric version is "bullseye", etc.